love/src/components/Facility/Map/Map.jsx

```diff
@@ -210,6 +211,12 @@ export default class Map extends Component {
     }
 
     const alarms_5 = [
+      dynaleneP05Events.dynMainGridAlarm ? dynaleneP05Events.dynMainGridAlarm.value : null,
+      dynaleneP05Events.dynMainGridFailureFlag ? dynaleneP05Events.dynMainGridFailureFlag.value : null,
+      dynaleneP05Events.dynSafetyResetFlag ? dynaleneP05Events.dynSafetyResetFlag.value : null,
+      dynaleneP05Events.dynTAalarm ? dynaleneP05Events.dynTAalarm.value : null,
+      dynaleneP05Events.dynTMAalarm ? dynaleneP05Events.dynTMAalarm.value : null,
+      dynaleneP05Events.dynaleneTankLevel ? (!dynaleneP05Events.dynaleneTankLevel.value === 0 ? true : false) : null,
```

Comment:
These lines could be written more shortly. E.g:
```javascript
const alarms_5 = [
  dynaleneP05Events?.dynMainGridAlarm?.value,
  dynaleneP05Events?.dynMainGridFailureFlag?.value
];
``` 
If you are not explicitly comparing the default `null` value, then this would do the same.

Reply:
For line 219:
```javascript
dynaleneP05Events?.dynaleneTankLevel?.value === 0
```